.pylintrc

```diff
@@ -432,6 +432,7 @@ disable=raw-checker-failed,
         too-few-public-methods,
         missing-module-docstring,
         logging-fstring-interpolation,
+        unused-argument,
```

Comment:
Not sure why this is disabled

Reply:
This was added as we made some placeholder methods that people can implement themselves:

`class DefaultResponseDsoService(ShapeshifterDsoService):
    def __init__(self):
        super().__init__(
            sender_domain=DSO_DOMAIN,
            signing_key=DSO_PRIVATE_KEY,
            key_lookup_function=key_lookup_function,
            endpoint_lookup_function=endpoint_lookup_function,
            port=DSO_TEST_PORT
        )

    def process_flex_offer(self, message):
        pass

    def process_flex_order_response(self, message):
        pass

    def process_d_prognosis(self, message):
        pass

    def process_flex_offer_revocation(self, message):
        pass

    def process_flex_settlement_response(self, message):
        pass

    def process_dso_portfolio_update_response(self, message):
        pass

    def process_dso_portfolio_query_response(self, message):
        pass

    def process_flex_request_response(self, message):
        pass

    def process_flex_reservation_update_response(self, message):
        pass

    def process_metering(self, message):
        pass`

Reply:
Hmm, I think file exclusions for this rule would be the best solution then, because in other places it is still valid. Not sure if this is a thing in pylint?

Reply:
I modified the code to disable the pylint warnings more specifically to methods where we know we can safely disable the warning instead of globally.